cmd/meroxa/root/apps/init.go

```diff
@@ -0,0 +1,94 @@
+package apps
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/log"
+	turbine "github.com/meroxa/turbine/init"
+)
+
+type Init struct {
+	logger log.Logger
+
+	args struct {
+		appName string
+	}
+
+	flags struct {
+		Lang string `long:"lang" short:"l" usage:"language to use (js|go)" required:"true"`
+		Path string `long:"path" usage:"path where application will be initialized (current directory as default)"`
+	}
+}
+
+var (
+	_ builder.CommandWithDocs    = (*Init)(nil)
+	_ builder.CommandWithArgs    = (*Init)(nil)
+	_ builder.CommandWithFlags   = (*Init)(nil)
+	_ builder.CommandWithExecute = (*Init)(nil)
+	_ builder.CommandWithLogger  = (*Init)(nil)
+)
+
+func (*Init) Usage() string {
+	return "init [APP_NAME] [--path pwd] --lang js|go"
```

Comment:
just thinking about ways to make the examples a little more copy and paste. feel free to ignore.
```suggestion
	return "init [APP_NAME] [--path $(pwd)] --lang js|go"
```